src/ipips/ipip-0431.md

```diff
@@ -46,11 +46,19 @@ HTTP client to opt-in via `Accept` header and Gateway to indicate via
 The proposed solution introduces a new parameter for the CAR content type in HTTP requests
 and responses: `meta`.
 
-When the CAR content type parameter `meta` is set to `eof`, the Gateway will write one additional CAR
-block with metadata to the response, after it sent all CAR blocks.
+The `meta` parameter allows clients to request the server to include additional metadata about the CAR along with the response body.
 
-The metadata format is DAG-CBOR and open to extension, allowing standardized
-userland experimentation similar to the Extensible Data field from IPNS V2.
+The value of this parameter includes both the location where the metadata is given (e.g. `eof`) as well as the type of data received (e.g. `json`) separated by a `+`, to give a value such as `meta=eof+json`
+
+When the location parameter is set to `eof`, which is currently the only supported value, the server SHOULD respond with <Response body as CARv1 stream> <0x00 byte> <Metadata>.
+
+The only supported value for the data type parameter is `json`. This signifies that the metadata MUST be of type `dag-json` (multicodec `0x0129`).
```

Comment:
I believe we are not using the multicodec `0x0129` anywhere in the spec, so this could be confusing. Can we remove the mention of that multicodec?

```suggestion
The only supported value for the data type parameter is `json`. This signifies that the metadata MUST be of type `dag-json`.
```

Reply:
I mustve missed this one as we want json not dag-json now. I will update.